File: Program3.java
// write  a program which accept number form user and display the below pattern

/*
 input : 7846

 output :       7  8  4  6
                7  8  4  
                7  8  
                7  
 */

import java.util.*;

public class Program3 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the length of the array: ");
        int n = sc.nextInt();

        System.out.println("Enter the array");
        int Arr[] = new int[n];
        for (int i = 0; i < n; i++) {
            Arr[i] = sc.nextInt();
        }
        for (int i = 0; i < n; i++) {
            int k = 0;
            for (int j = 0; j < (n - i); j++) {
                System.out.print(Arr[k] + "\t");
                k++;
            }
            System.out.println();
 }
}
}

File: program3.c
// Accept two values and perform addition

// Step 1  : Understand the problem statement
// conclusion : we have to accept two integers and perform its addition


// step 2 : Write the alogorithm
/*
    START
        Accept First Number from user and store it into No1
        Accept second number from user and store it into No2
        Create one varibale tp store the result name as answer
        Perform addition of No1 and No2, store the result in Ans
        Display the result from Ans to the user
    STOP 
*/

// Step 3 : Decide the programming language ( C, C++ , Java)
// we select C programming


// Step 4 :  Write the program



#include<stdio.h>

int main()
{

   int a;
   int b;
   int addition;

   printf("Enter number 1 : ");
   scanf("%d",&a);

   printf("Enter number 2 : ");
   scanf("%d",&b);
 
   addition = a + b;

   printf("The Addition is : %d ",addition);


    return 0;
}